### ISSUE TITLE
Markdown Issue

### ISSUE THREAD

**Issue by crimsenraven** [securestate/king-phisher#324]
I get the following error when trying to start the KingPhisher client

Traceback (most recent call last):
  File "/opt/king-phisher/KingPhisher", line 48, in <module>
    from king_phisher.client import application
  File "/opt/king-phisher/king_phisher/client/application.py", line 56, in <module>
    from king_phisher.client import dialogs
  File "/opt/king-phisher/king_phisher/client/dialogs/__init__.py", line 35, in <module>
    from .clone_page import *
  File "/opt/king-phisher/king_phisher/client/dialogs/clone_page.py", line 37, in <module>
    from king_phisher.client.widget import extras
  File "/opt/king-phisher/king_phisher/client/widget/extras.py", line 46, in <module>
    import mdx_partial_gfm
  File "/usr/local/lib/python3.6/dist-packages/mdx_partial_gfm/__init__.py", line 33, in <module>
    import gfm
  File "/usr/local/lib/python3.6/dist-packages/gfm/__init__.py", line 15, in <module>
    from gfm import spaced_link
  File "/usr/local/lib/python3.6/dist-packages/gfm/spaced_link.py", line 38, in <module>
    BRK = markdown.inlinepatterns.BRK
AttributeError: module 'markdown.inlinepatterns' has no attribute 'BRK'

I have remove Markdown 3.1 and installed 2.6.8 but the issue persists.

Thank you.

**Comment by zeroSteiner** [comment]
What about version 2.6.11 of the markdown package?

**Comment by crimsenraven** [comment]
Same error.

**Comment by zeroSteiner** [comment]
Can you post the output of `python3 -m pip freeze`?

**Comment by routeback** [comment]
Can confirm I experienced this issue as well, even after reverting to 2.6.11.
Still using version King Phisher version 1.11.1 for stability: git reset --hard 4af4fdc

**Comment by zeroSteiner** [comment]
Thanks, yeah we're working on a fix for this, but it'll be a few days.

**Comment by zeroSteiner** [comment]
So the issue looks like a compatibility issue between the py-gfm and markdown packages. I can fix it on my system by downgrading the markdown package to `2.6.11` using `python3 -m pip install 2.6.11`. I'm not sure why installing that version hasn't worked for others though.

**Comment by zeroSteiner** [comment]
Alright so this has been fixed in master and dev now. I'm highly confident that the issue lies with the incompatibilities between the two markdown packages. The solution I implemented pins the markdown package to version `2.6.11`.

Go ahead and upgrade to master and reinstall the requirements file with `python3 -m pip install -r requirements.txt` (this is the same as the last step in the [updating instructions](https://github.com/securestate/king-phisher/wiki/Updating-King-Phisher)). If it was experiencing this issue before, this will downgrade your markdown package.
